binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -128,6 +132,14 @@ public void run() {
                                 }
                             }
                         }
+                        
+                        final Config config = fullConfig.getConfig(); 
+                        if (config != null) {
+                            storeConfigValue(getThing().getConfiguration(), DefaultPropertyKey.SERIAL_NUMBER,
+                                    config.getMACAddress().replaceAll(":", ""));
```

Comment:
Because this was part of the acceptance criteria defined by the PO. I just recognize one bug: The acceptance criteria defines among others the following:
id=("mac" address of bridge without : and fffe between) from Philips REST API (e.g. "001788fffe169b87"). This is the id also returned by Network UPnP call at https://www.meethue.com/api/nupnp
serialNumber="<serial-number-from-device>" (same is id)

So I have forgotten the fffe in between... And because of the id parameter has not been fully discussed yet I put it as serial number value...as required...